### PR TITLE
Updates to HCAL relval scripts and standalone configs

### PR DESCRIPTION
**Pull request by pastika**
Changes in  Validation/CaloTowers/test
- RelValHarvest.py updated to circumvent poorly named relval dataset status files
- new configs for rerunning DQM file feneration on data added

Is it still possible to include this change in a 7_4_X release as well as well?

### CODE REVIEW

**Comment by deguio** [comment]
+1

File: Validation/CaloTowers/test/run_onRelVal_data_cfg.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
+import os
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("hcalval")
+process.load("Configuration.StandardSequences.Reconstruction_cff")
+process.load("Configuration.StandardSequences.Geometry_cff")
```
Comment (davidlange6) [inline Validation/CaloTowers/test/run_onRelVal_data_cfg.py]: This is the wrong geometry file to load. Please replace it with GeometryRecoDB_cff instead.

Comment (pastika) [inline Validation/CaloTowers/test/run_onRelVal_data_cfg.py]: We will be preparing another pull request for further HCAL relval 
changes in the near future.  We will address this in this pull request.

Joe

Comment (davidlange6) [inline Validation/CaloTowers/test/run_onRelVal_data_cfg.py]: would be great just to update the current branch - then it won't be forgotten (and that update will automatically enter this PR). Thanks.

Comment (pastika) [inline Validation/CaloTowers/test/run_onRelVal_data_cfg.py]: Change to branch made.

**Pull request merged**